app/models/middleware_server_group.rb

```diff
@@ -1,6 +1,9 @@
 class MiddlewareServerGroup < ApplicationRecord
   belongs_to :middleware_domain, :foreign_key => "domain_id"
   has_many :middleware_servers, :foreign_key => "server_group_id", :dependent => :destroy
+  has_many :middleware_deployments, :foreign_key => "server_group_id", :dependent => :destroy
   serialize :properties
   acts_as_miq_taggable
+
+  delegate :ext_management_system, to: :middleware_domain, :allow_nil => true
```

Comment:
My bad, I shouldn't blindly copy&paste the code of questionable quality from the internets :grinning: